Game/Game.js

```diff
@@ -1,45 +1,63 @@
 class Game {
-    constructor(wordToGuess, maxWrongAttempts) {
+    constructor(wordToGuess, maxWrongAttempts = 6) {
         // this.alive = true;
         this.player1;
         this.player2;
-        this.wordToGuess = wordToGuess;
+        this.wordToGuess = wordToGuess.toLowerCase();
         this.maxWrongAttempts = maxWrongAttempts;
         this.attemptedGuesses = []; // All of these will be displayed as guesses
         this.wrongGuesses = 0;
         this.correctGuesses = []; // These will be displayed as correct guesses only
     }
+
     reviewAttempt(guess) {
+        guess = guess.toLowerCase();
+        if (this.attemptedGuesses.includes(guess)) {
+          return;
+        }
         this.attemptedGuesses.push(guess);
         if (this.wordToGuess != guess && !this.wordToGuess.includes(guess)) {
-            this.wrongGuesses += 1;
-            this.wrongAttempt();
+            return this.wrongAttempt(guess);
         } else {
             this.correctGuesses.push(guess);
-            this.correctAttempt();
+            return this.correctAttempt();
         }
     }
-    wrongAttempt() {
+
+    wrongAttempt(guess) {
+        this.wrongGuesses += 1;
         if (this.wrongGuesses >= this.maxWrongAttempts) {
-            alert('The man is dead');
-            location.reload();
```

Comment:
Did we get rid of location.reload() all together? I guess it wasn't there when it got implemented, but just adding a comment to reference for a final sweep of this.

Reply:
Totally. I think we covered this in person, but just for documentation sake:

Since we moved this file over to the server-side, it will be running in a node.js environment now instead of in the window. However, we can totally link up the responses to `location` actions on the client-side again!